plugins/vpp/l3plugin/vppcalls/vpp1904/dhcpproxy_dump.go

```diff
@@ -0,0 +1,59 @@
+package vpp1904
+
+import (
+	"fmt"
+	vpp_l3 "github.com/ligato/vpp-agent/api/models/vpp/l3"
+	"github.com/ligato/vpp-agent/plugins/vpp/binapi/vpp1904/dhcp"
+	"github.com/ligato/vpp-agent/plugins/vpp/l3plugin/vppcalls"
+	"net"
+)
+
+func (h *DHCPProxyHandler) DumpDHCPProxy() (*vppcalls.DHCPProxyDetails, error) {
+	var entry *vppcalls.DHCPProxyDetails
+	reqCtx := h.callsChannel.SendMultiRequest(&dhcp.DHCPProxyDump{})
+	for {
+		dhcpProxyDetails := &dhcp.DHCPProxyDetails{}
+		stop, err := reqCtx.ReceiveReply(dhcpProxyDetails)
+		if stop {
+			break
+		}
+		if err != nil {
+			h.log.Error(err)
+			return nil, err
+		}
+
+		var srcIP string
+		if dhcpProxyDetails.IsIPv6 == 1 {
+			srcIP = fmt.Sprintf("%s", net.IP(dhcpProxyDetails.DHCPSrcAddress).To16().String())
```

Comment:
You can omit `fmt.Sprintf` function here and just use return value from `String()`. Same for else case.